package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "business-create-ui",
-  "version": "5.5.8",
+  "version": "5.6.0",
```

Comment:
This is the start of code changes for amalgamation filings so I increased the minor version number.